### DISCUSSION TITLE
Light Groups with Sequent 8 relay

### DISCUSSION THREAD

**Discussion by truvec** [tagyoureit/nodejs-poolController#880] [Nixie]
I have 6 lights set up through REM relays. Aux circuit control of each light is effective.  But when using a Light Group for them the toggle only turns of the first light(relay) in REM. The dashboard indicate lights off but it doesnt get to REM. It also doesnt turn off the 6th light on the dashboard(if the circuit is off the Group toggle will turn it on). 
<img width="397" alt="Screenshot 2024-01-11 at 11 08 35 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/239790/b522b57e-0352-4635-8bae-e01653ef969f">
<img width="392" alt="Screenshot 2024-01-11 at 11 08 24 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/239790/5d25cc56-ebe8-4c57-a196-edb3627a21ab">
<img width="715" alt="Screenshot 2024-01-11 at 10 58 07 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/239790/b2794898-1450-499f-930a-4fbeb7ff77d7">

**Comment by tagyoureit** [comment]
Hopefully fixed per #881.
